src/files-and-videos/assets/index.js

```diff
@@ -0,0 +1,3 @@
+import FilesAndUploads from './FilesAndUploads';
```

Comment:
This file is only associated with the Files page while FileMenu is used by both files and videos

Reply:
Files is the generic designation for both assets and videos. Assets is the route for the files page, so that I gave those related files the same folder name.